src/test/java/org/openhmis/test/BaseIntegrationTest.java

```diff
@@ -0,0 +1,127 @@
+package org.openhmis.test;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Properties;
+
+import javax.ws.rs.core.Application;
+import javax.ws.rs.core.Response;
+
+import org.dbunit.IDatabaseTester;
+import org.dbunit.JdbcDatabaseTester;
+import org.dbunit.database.IDatabaseConnection;
+import org.dbunit.dataset.IDataSet;
+import org.dbunit.operation.DatabaseOperation;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.test.JerseyTest;
+import org.hibernate.SQLQuery;
+import org.hibernate.Session;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openhmis.util.ApplicationPropertyUtil;
+import org.openhmis.util.HibernateSessionFactory;
+import org.openhmis.webservice.ClientService;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.openhmis.dto.result.ListResultDTO;
+import org.openhmis.provider.ObjectMapperContextResolver;
+import org.openhmis.provider.WrappingWriterInterceptor;
+import org.openhmis.dto.ClientDTO;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import org.flywaydb.core.Flyway;
+
+public class BaseIntegrationTest extends JerseyTest {
+
+	private Session session; 
+	private Properties properties;
+	
+    @Override
+    protected Application configure() {
+    	
+    	// Spread the word that this is a test
+    	// TODO: This isn't the right way to do this.
```

Comment:
Can you expound on that?  Do we have some vague idea of what the right way would be?  (I'm not against leaving the "TODO" in the code for now, but I'd like to know whether we should file an issue, and what that issue should say, etc.)